types/enums.ts

```diff
@@ -8,3 +8,7 @@ export enum STORAGE_KEY {
   CONNECTION_EXPIRY = 'LUKSO_WALLET:connectionExpiry',
   RECONNECT_ADDRESS = 'LUKSO_WALLET:reconnectAddress',
 }
+
+export enum REFERRERS {
+  INDEXER = 'indexer',
+}
```

Comment:
Maybe a little comment to add context:

```suggestion
// We use ?referrer=XXX in the URL queries to enable/disable specific features
export enum REFERRERS {
  INDEXER = 'indexer',
}
```